feathers-query-logic.js

```diff
@@ -14,6 +14,12 @@ module.exports = {
 		}
 		query.filter = filter;
 
+		var filterProp = Object.keys(query.filter);
+		filterProp.forEach(item => {
```

Comment:
This arrow function needs to be the “normal” function syntax because we support IE11.

Reply:
@chasenlehara Done, I think... lol Normal as in `function(item) { ... }` right?

Reply:
Yup, thanks!

Would it be possible to add tests for these changes here? https://github.com/canjs/feathers-query-logic/blob/master/feathers-query-logic-test.js

Reply:
Sure.